src/taipy/core/cycle/_cycle_manager.py

```diff
@@ -15,26 +15,23 @@
 
 from taipy.config.common.frequency import Frequency
 
+from .._entity._entity_ids import _EntityIds
 from .._manager._manager import _Manager
-from ..common._entity_ids import _EntityIds
-from ..common.alias import CycleId
 from ..job._job_manager_factory import _JobManagerFactory
 from ._cycle_repository_factory import _CycleRepositoryFactory
 from .cycle import Cycle
+from .cycle_id import CycleId
 
 
 class _CycleManager(_Manager[Cycle]):
     _repository = _CycleRepositoryFactory._build_repository()  # type: ignore
     _ENTITY_NAME = Cycle.__name__
 
     @classmethod
-    def _create(
```

Comment:
Hi @jrobinAV 
Is there a reason why you removed the display_name parameters? It's causing some trouble in the enterprise version :/